### PR TITLE
Completed the PrivateWikis feature

### PR DESCRIPTION
**Pull request by praveenyr**
**Description :** Completed the Private Wikis feature and tested successfully.

**Test Coverage :** 88.27%

### CODE REVIEW

**Comment by praveenyr** [comment]
@wonderwoman13 I forgot to mention that I created a 'All wikis' link on the nav bar which basically lists all the public wikis.Whats the point of a wiki site if a user cant see the all the public wikis.Silly me :)

File: app/views/wikis/index.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="row">
-   <div class="col-md-8">
+   <div class="col-md-10">
      <h2 class="text-center" style="margin: 20px;"><u>My wikis</u></h2>
      <% if policy_scope(@wikis).count > 0 %>
```
Comment (brittanyjspringer) [inline app/views/wikis/index.html.erb]: @praveenyr the only change I'm going to request is for you to handle the policy_scope in the controller, not the view. I need clarification why you would wrap @wikis in a policy_scope when you did so in the controller already.

File: app/controllers/wikis_controller.rb [hunk 2]
```diff
@@ -5,6 +5,11 @@ class WikisController < ApplicationController
   def index
     @wikis = policy_scope(Wiki)
   end
+  
+  def public_wikis
```
Comment (brittanyjspringer) [inline app/controllers/wikis_controller.rb:9]: I like this idea!

**Review by brittanyjspringer — CHANGES_REQUESTED** [review]

File: app/views/wikis/index.html.erb (hunk 1 above)
Comment (praveenyr) [reply app/views/wikis/index.html.erb]: @wonderwoman13 Nothing specific to clarify,it was a copy paste :).I removed the policy scope from the  index html.

**Review by brittanyjspringer — APPROVED** [review]

**Comment by brittanyjspringer** [comment]
Great! Merged. Will approve your checkpoint.

**Pull request merged**